### PR TITLE
add an option to install charts from a locally mounted volume in addi…

### PR DESCRIPTION
**Pull request by ngoracke**
…tion to oci image

Signed-off-by: Nicholas Goracke <ngoracke@us.ibm.com>

### CODE REVIEW

**Comment by ngoracke** [comment]
We need the ability to install charts that are not available in a non-authenticated registry.  This gives the option to provide a PVC populated with charts.

**Comment by roee88** [comment]
If there was support for authenticated registries would you still need this?

**Comment by ngoracke** [comment]
@roee88 This is still quite convenient for development - it allows us to dump charts on volume, tweak as needed, and submit fybrik applications to exercise without publishing to an authenticated registry each iteration.

So I would say: yes.

**Comment by ngoracke** [comment]
This *may* also be useful for air-gapped scenarios, where a registry that supports helm-charts-as-images is not available, and there is no desire to set up a separate helm chart repository.  That's debate-able, though.

File: charts/fybrik/values.yaml [hunk 1]
```diff
@@ -171,6 +171,8 @@ manager:
 
   prometheus: false
 
+  chartVolume: ""
```
Comment (roee88) [inline charts/fybrik/values.yaml:174]: Consider adding a comment on how to use it.

**Review by roee88 — APPROVED** [review]

**Comment by roee88** [comment]
> @roee88 This is still quite convenient for development - it allows us to dump charts on volume, tweak as needed, and submit fybrik applications to exercise without publishing to an authenticated registry each iteration.
> 
> So I would say: yes.

I think that an in cluster registry would be a better approach, I believe that we have used it successfully it in the past. Anyway, this PR looks good for me as a short term approach. The issue for adding authentication support is still open as a longer term goal. 

+ @revit13 for awareness because these changes need to be considered when upgrading to Helm 3.7.

File: charts/fybrik/values.yaml (hunk 1 above)
Comment (ngoracke) [reply charts/fybrik/values.yaml:174]: Done

**Comment by ngoracke** [comment]
Just a bit more detail on the last comment... We had done some experimentation with an in-cluster and out-of-cluster registry, but had issues with: https://github.com/helm/helm/issues/6324

Summary: the default communication protocol is https when not working with localhost:5000, and the operations we require don't support the --insecure options.  I believe we are getting around this with the KIND environment because we have configuration to route kube traffic to localhost:5000 to kind-registry:5001 running in docker, but this would likely not be something we could count on for a real kube cluster.  In other words, we'd require that in-cluster registry to be running https, and have valid non-self-signed certificates.

**Review by revit13 — APPROVED** [review]

**Pull request merged**